questions/191/explanation.md

```diff
@@ -2,5 +2,5 @@ Due to the `extern "C"` specifications, `A::x` and `B::x` actually refer to the
 
 `x` is first initialized to `0`, then `main()` starts, `0` is printed, `x` is incremented to `1`, and finally `1` is printed.
 
-§[dcl.link]¶6 in the C++ standard:
-"Two declarations for a variable with C language linkage with the same name (ignoring the namespace names that qualify it) that appear in different namespace scopes refer to the same variable."
+§[dcl.link]¶7 in the C++23 standard:
```

Comment:
Let's use just "C++" here without specifying the exact version. See https://github.com/knatten/cppquiz23/pull/201#discussion_r1658104656.